client/app/components/packages/pas-form/edit.js

```diff
@@ -57,26 +58,32 @@ export default class PasFormComponent extends Component {
       isApplicantsDirty,
     } = this.pasForm;
 
-    return (isPasFormDirty && isPasFormValid) || isBblsDirty || isApplicantsDirty;
+    return !this.isUpdatingDb && ((isPasFormDirty && isPasFormValid) || isBblsDirty || isApplicantsDirty);
```

Comment:
cool :) Thanks. later, we can really easily split this out :)

Reply:
🙌 🙌